### PR TITLE
Release3 cleanup

### PR DESCRIPTION
**Pull request by brian-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

Refactoring code to only include content related to the automated self checkout use case. Performance tool submodule has been added to provide benchmark features.

## Issue this PR will close

close: #**issue_number**

## Test Instructions if applicable

- Make run-demo
- See video stream with full pipline running. This includes object detection, text detection, text recognition, and barcode detection.

### CODE REVIEW

File: src/cl-cache/cache.txt
Comment (jim-wang-intel) [inline src/cl-cache/cache.txt:1]: remove: should not check-in this file

**Review by jim-wang-intel — APPROVED** [review]

**Review by jim-wang-intel — APPROVED** [review]
LGTM!

**Review by NeethuES-intel — COMMENTED** [review]
Tested demo, worked, however video didn't pop up.  
We do need to update the ReadMe, separate PR could be created to cover these changes -
- Update prereq - git version > 2.30
- Detailed instructions under Quick start need to be updated based on latest changes 
- smoke test doesn't work, need to either remove or update steps in ReadMe
- should include `make stop-demo` after the `make run-demo` step

If release is not advertised, then we can update this later along with documentation.

**Review by NeethuES-intel — APPROVED** [review]
LGTM, everything worked. I do see lag in video, probably my i5 machine is not able to handle processing

**Pull request merged**